src/lib/index.js

```diff
@@ -344,6 +344,17 @@ lib.mergeArray = function(traceAttr, cd, cdAttr) {
     }
 };
 
+lib.getTargetArray = function(trace, target) {
+    if(typeof target === 'string' && target) {
+        var array = lib.nestedProperty(trace, target).get();
+
+        return Array.isArray(array) ? array : [];
```

Comment:
> in these applications, of what we should do when targetArray isn't the same length as the other associated arrays. Looks like for both, it will truncate (or extend) all of them to the length of targetArray. Is that what we want? Seems right for sort, but if we agree

Yep, that's how I designed back when `filter` was first added. 

> but if we agree then lets :lock: it with a test. 

Sure thing.

>  can you give the function a docstring?

good call :+1: